src/components/budget/budget-record.repository.ts

```diff
@@ -136,12 +136,13 @@ export class BudgetRecordRepository extends DtoRepository(BudgetRecord) {
           ])
           .apply(matchChangesetAndChangedProps(changeset))
           .apply(matchPropsAndProjectSensAndScopedRoles(session))
-          .return<{ dto: UnsecuredDto<BudgetRecord> }>(
+          .return<{ dto: UnsecuredDto<BudgetRecord>; scope: ScopedRole[] }>([
             merge('props', 'changedProps', {
               organization: 'organization.id',
               changeset: 'changeset.id',
-            }).as(outputVar)
-          )
+            }).as(outputVar),
+            'scopedRoles as scope',
+          ])
```

Comment:
I think you are probably behind develop. BudgetRecord already has a scope array. https://github.com/SeedCompany/cord-api-v3/blob/develop/src/components/budget/budget-record.repository.ts#L140-L145

Reply:
Yeah, I did a bunch of crazy stuff like that then realized we want to cache the scope in the dto. the next push should fix most of that.